meson.build

```diff
@@ -0,0 +1,18 @@
+  project('cpputest', 'cpp',
+    version : '4.0',
+    license : 'BSD-3')
+
+cpputest_dirs = include_directories('include')
+subdir('src')
+
+cpputest_dep = declare_dependency(
+    link_with : cpputest_lib,
+    version : meson.project_version(),
+    include_directories : cpputest_dirs)
+
+if get_option('extinctions') == true
```

Comment:
`if X == true` can generally be rewritten as `if X` I believe and it looks much better.